templates/letsencrypt/cert-manager.yaml

```diff
@@ -0,0 +1,8399 @@
+
```

Comment:
This looks like another big file that we could install from a URL instead of making our own copy if we're not having to modify it at all.

Reply:
That sounds like a good idea. Some of these URLs aren't versioned, but it looks like this one is. Perhaps let's merge this as-is for now (to give existing projects an easy migration path from the hosting-services repo) and then the next time we upgrade cert-manager we can switched to a URL to simplify updating it in the future.